packages/cli/test/services/environment.test.ts

```diff
@@ -21,7 +21,6 @@ describe('environment service', (): void => {
   describe('currentEnvironment', (): void => {
     it('get current environment: testing', async () => {
       process.env.BOOSTER_ENV = 'testing'
-      stub(process.env, 'BOOSTER_ENV').value('testing')
```

Comment:
We shouldn't set environment variables in unit tests, especially if we don't restore them in an after-block. Maybe we could use here the `replace` and `restore` functions from `sinon` to avoid setting the actual variable and ensure the changes are reset afterward. I think that something like this should work:

```typescript
  ...
  beforeEach(() => {
    replace(process.env, 'BOOSTER_ENV', 'testing')
  }

  afterEach(() => {
    restore()
  }
  ...
```

Reply:
I agree, test changed but at least we need to create the variable empty to being able to replace it later.

Are you okey with that?

```
...
beforeEach(() => {
        process.env.BOOSTER_ENV = ''
        replace(process.env, 'BOOSTER_ENV', 'testing')
      })

      afterEach(() => {
        restore()
      })
...
```

Reply:
Oh I see T_T

If that's the case, there's not much difference between setting it to `''` or any other value.

Anyways, inspecting the code again, I found that we were already handling the environment variables state in this code block between lines 7 and 19:

```typescript
  beforeEach(() => {
    if (process.env.BOOSTER_ENV !== undefined) {
      formerValue = process.env.BOOSTER_ENV
    }
    delete process.env.BOOSTER_ENV
  })
  afterEach(() => {
    process.env.BOOSTER_ENV = formerValue
    restore()
  })
```

With this code in place, I'd say that your initial solution of setting the BOOSTER_ENV value without using `replace` and `restore` was correct. Sorry about that. I'll add a commit to your branch removing the unnecessary calls to these methods